### PR TITLE
Add Jira issue vendor

### PR DESCRIPTION
**Pull request by ruizb**
**URL**
Pattern: `/jira/issue/<protocol>/<host>/<issue-key>.<format>`
Example: `/jira/issue/https/issues.apache.org/jira/KAFKA-2896.svg`

**Auth**
Add `jira_username` and `jira_password` entries to the *./secret.json* file.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -284,6 +284,63 @@ camp.notfound(/.*/, function(query, match, end, request) {
 
 // Vendors.
 
+// JIRA integration
+camp.route(/^\/jira\/(http(?:s)?)\/(.+)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```
Comment (espadrine) [inline server.js]: Could you add a badge-specific prefix, in case we have multiple JIRA badges in the future? Maybe `jira/issue`?

**Comment by espadrine** [comment]
Thanks a lot for your contribution!

File: server.js (hunk 1 above)
Comment (ruizb) [inline server.js]: You are right, I'll make the change ASAP. Thanks for the feedback!

**Comment by ruizb** [comment]
I added `issue`in the prefix in case there are more JIRA routes in the future.
Thank you for the feedback!

**Comment by espadrine** [comment]
Thanks again! It is now live, and the website will be updated in 24h.

**Pull request merged**